### DISCUSSION TITLE
Say Hello to Obsrv!

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Obsrv/Community#148] [Announcements]
How often have you wished for a Genie that can get you all the data you need, exactly when you want it, in the form and shape that you wanted it in? 
We give you Obsrv (well, it doesn't fit into a lamp yet, but let's say we're working on it) 

A cutting-edge, low-code next generation Data Platform, Obsrv will reshape the landscape of data management as you know it. The single largest offering of its kind, Obsrv offers reliable, scalable and flexible data capabilities - which will ensure that all your data needs are met, at your specific scale requirements. It handles every aspect of the data lifecycle – from seamless ingestion and efficient processing to robust storage and lightning-fast, near real-time querying and reporting. What sets Obsrv apart is its commitment to providing unparalleled performance, flexibility and ease of usage, along with the freedom of choice it offers. 

We will host an hour-long webinar that will cover all key aspects of the Obsrv offering - easy installation, the wealth of capabilities it offers, sizing and infra needs, as well as address any other questions you may have. 

Date & Time : Thursday, 15th February 2024 
Registration link : https://zoom.us/webinar/register/WN_Faxs5kZkSNGWonfmSBkfiQ 

![Obsrv](https://github.com/Sunbird-Obsrv/Community/assets/24287865/9ff21410-2206-45a5-835a-082ab722c880)

Register now and wish away your data worries. Craft the data experience of the future - your Genie awaits! 

Best, 
Team Obsrv

**Comment by rhwarrier** [comment]
Thank you for your interest in Obsrv - the one stop solution to all your data needs. We hosted an engaging webinar last evening to introduce Obsrv to an audience of over 140 enthusiastic participants. We had fun detailing the Obsrv offering, and responding to queries and clarifications. It was indeed heartening to see such interest in a product that is being honed to work seamlessly across a large spectrum of domains. Your questions and inputs will go a long way to help us continuously make Obsrv better in the days to come.  

Here are some links and assets you may find useful in the context of Obsrv:

The questions (and answers) raised during the webinar will be posted on the Obsrv Discussion Forum. Please use the forum to raise any other questions you may have - they will be responeded to, promptly.

Follow the [Obsrv Linkedin page](https://www.linkedin.com/company/97450008/) to stay in touch, and find out more about upcoming events

You can write to connect@obsrv.ai if you'd like to try out a Sandbox experience, or need help with Obsrv set up.

Follow us on Linkedin to ensure you don't miss out on announcements and exciting events that are lining up. 
Here's to growing the Obsrv community with all of you - and transforming the data landscape as we know it!

Best,
Team Obsrv

**Reply by vrayulu** [reply]
The webinar recording is available at https://youtu.be/AeUIW0lVV1g?feature=shared
